node-operators/networks/run-a-node/overview.md

```diff
@@ -11,11 +11,11 @@ description: 了解为Moonbeam网络运行平行链全节点以便您拥有自
 
 Moonbeam拥有多种不同的部署渠道，包含Moonbase Alpha测试网，Kusama上的Moonriver，以及波卡（Polkadot）上的Moonbeam。以下是网络的环境名称以及其对应的[区块链参数文档](https://substrate.dev/docs/en/knowledgebase/integrate/chain-spec)名称：
 
-|      网络      |  |  托管方   |  |             区块链名称              |
-|:--------------:|--|:---------:|--|:-----------------------------------:|
-| Moonbase Alpha |  | PureStake |  | {{ networks.moonbase.chain_spec }}  |
-|   Moonriver    |  |  Kusama   |  | {{ networks.moonriver.chain_spec }} |
-|    Moonbeam    |  | Polkadot  |  | {{ networks.moonbeam.chain_spec }}  |
+|      网络      |  |       托管方        |  |             区块链名称              |
```

Comment:
can you please update this table so it doesn't use the two pipes to separate the columns (`|  |`)?